lib/Logger/LoggerFeature.cpp

```diff
@@ -96,6 +96,7 @@ LoggerFeature::~LoggerFeature() {
 
 void LoggerFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
   options->addOldOption("log.tty", "log.foreground-tty");
+  options->addOldOption("log.escape", "log.escape-control-chars");
```

Comment:
If the old option acts as an alias, then I suppose we cannot `.setDeprecatedIn(30900)`?

Reply:
Correct, `addOldOption(...)` returns `void`, so we cannot add a deprecation here.
I think what it does internally is just add an entry to an "old name" => "new name" map.